Minion needs to be restarted after bootstrap (#530)
SUSE/doc-susemanager: modules/administration/pages/virtualization.adoc

** Execute the bootstrap script to register the {vmhost}.
+

[IMPORTANT]
====

> After bootstrap, Salt minions need to be restarted in order to make virtualization fully work.

Reviewer: Would rather write:

    in order to get instant virtual machines changes reflected in {productname}

Reviewer 2: see my recent sugggestion at the end of this thread.

> After bootstrap, Salt minions need to be restarted in order to get instant virtual machines changes reflected in {productname}.

Reviewer 2: ```suggestion
After bootstrap, Salt minions need to be restarted in order to get instant virtual machine changes reflected in {productname}.
```
Sounds better, IMO (but could be wrong).

Reviewer 3: I'll have a go, too ;)

```suggestion
Salt minions must be restarted after bootstrapping, in order for instant virtual machine changes to be reflected in {productname}.
```

Reviewer: s/bootstrapping/enabling Virtualization Host entitlement/ would be more accurate

> Salt minions must be restarted after enabling the Virtualization Host entitlement in order for instant virtual machine changes to be reflected.

Reviewer 2: I reworked Cedric's proposal as a suggestion:
```suggestion
Salt minions must be restarted after enabling the Virtualization Host entitlement in order to see virtual machine changes in {productname} instantly.
```

Reviewer: @keichwa LGTM
